arm: dts: aspeed-g7: update i2c clock speed to 400khz

aasboddu opened the pull request:
Modified i2c clock speed to 400khz for i2c-7,8,9,10.

Tested:- Verified on Congo and Morocco Platforms.

aasboddu committed:
ARM64: dts: aspeed: Congo: Update i2c clock to 400khz

Update i2c buses to support 400khz clock speed.

Tested: Verified the i2c buses for Congo

Signed-off-by: aasboddu <Aashray.Boddu@amd.com>

mahkurap reviewed (changes requested):
instead of changing the dtsi file, overwrite this property in the corresponding device tree file.

aasboddu committed:
ARM64: dts: aspeed: Morocco: Update i2c clock to 400khz

Update i2c buses to support 400khz clock speed.

Tested: Verified the i2c buses for Morocco

Signed-off-by: aasboddu <Aashray.Boddu@amd.com>

aasboddu commented:
> instead of changing the dtsi file, overwrite this property in the corresponding device tree file.

Moved changes to dts file. Created individual commits for each platform.
